coins/zerc.json

```diff
@@ -7,6 +7,8 @@
         "K": 7,
         "personalization": "ZERO_PoW"
     },
+    "sapling": 501000,
+    "overwinter": 501000,
```

Comment:
Can you remove `overwinter`? If sapling is active, so is overwinter. Also, if ZERC is now passed block 501000, can you set this to true? Thanks!

Reply:
Changed accordingly. Please review. Thanks.